### ISSUE TITLE
False "field 'x' is never used" warnings

### ISSUE THREAD

**Issue by Jobots** [SaladLab/Unity3D.IncrementalCompiler#3]
Hello,

First of, thank you for making this awesome thing!

This is kind of a nitpick, but wanted to report it anyway. I'm getting a few CS0169 "field 'x' is never used" warnings on fields that definitely are used.

Here's an example - I'm getting CS0169 for the warpEngine field:

```
using UnityEngine;
using System.Collections;
using System;

namespace palimon
{
    public class WarpEngineController : Controller
    {
        private WarpEngine warpEngine;

        private void Awake()
        {
            warpEngine = GetComponent<WarpEngine>();
            if (warpEngine == null) throw new Exception("WarpEngineController unable to find adjoining WarpEngine component.");
        }

        public override void Operate(InputQueue input)
        {
            base.Operate(input);

            if (input.GetKeyUp(PilotAction.WarpJump))
            {
                if (warpEngine.isReadyToSpoolUp) warpEngine.SpoolUp();
                else if (warpEngine.isWarping && warpEngine.isReadyToExitWarp) warpEngine.ExitWarp();
            }
        }
    }
}
```

But I'll happily take a few false warnings for 4x faster compilation times. :)

**Comment by veblush** [comment]
Thanks for reporting. I just forgot passing `nowarn` options to Roslyn compiler from Unity.
It's fixed in [1.2.1](https://github.com/SaladLab/Unity3D.IncrementalCompiler/releases/tag/1.2.1).
